docker/Dockerfile

```diff
@@ -0,0 +1,116 @@
+FROM --platform=linux/amd64 node:16.13.0 AS frontend
```

Comment:
yes, as per our discussion over Chat, this env variable is specifically required so that the server runs properly on macOS (silicon chip) in which we require to specifically set the platform for the built images where the server will  be running. refer to the first answer [here](https://stackoverflow.com/questions/71040681/qemu-x86-64-could-not-open-lib64-ld-linux-x86-64-so-2-no-such-file-or-direc) basically we are defining the infrastructure as code for the containers where the server will be running.


also, note that this platform flag specified the platform for the built output i.e the images/containers where our server will be running. refer [here](https://docs.docker.com/build/building/multi-platform/#:~:text=When%20you%20invoke%20a%20build,can%20specify%20multiple%20platforms%20together.)